mpt/impl/managesieve/core/src/main/resources/org/apache/james/managesieve/scripts/capability.test

```diff
@@ -39,7 +39,7 @@ S: OK
 
 C: AUTHENTICATE "PLAIN"
 S: \+ ""
-C:  user password
+C: "user password"
```

Comment:
When authentication happens as part of continuation I am rather sure that the crediential must nt be quoted


```suggestion
C: user password
```

CF https://datatracker.ietf.org/doc/html/rfc4616#section-4